docs/patterns.json

```diff
@@ -11088,7 +11215,7 @@
         },
         {
           "name": "langDir",
-          "default": "/Users/heliorocha/dev/codacy/codacy-eslint/node_modules/eslint-plugin-spellcheck/rules/utils/dicts"
```

Comment:
we should later try to understand why this value is hardcoding local information from the dev machine.